website/docs/cloud-docs/workspaces/health.mdx

```diff
@@ -95,17 +95,21 @@ On the right of a workspace’s overview page, Terraform Cloud displays a **Heal
 
 ## Drift Detection
 
-Infrastructure drift means that your real-world infrastructure no longer matches your Terraform state file. Drift occurs when a user modifies resources outside of the Terraform workflow. For example, a colleague may update resource configuration directly in the cloud provider console to resolve a production incident. This action changes the real resource attributes from those tracked in the state file.
+Drift detection helps you identify situations where your actual infrastructure no longer matches the configuration you've defined in Terraform. This deviation is known as configuration drift. Configuration drift occurs when changes are made outside of Terraform's regular process, leading to inconsistencies between the remote objects and your configured infrastructure.
+
+For instance, if a colleague makes direct updates to remote objects using your cloud provider's console to address a production issue, the changes they make might conflict with the intended configuration. In such cases, Drift Detection detects these differences and recommends steps to address and rectify the discrepancies.
+
+~> **NOTE:** Configuration drift is distinct from state drift, which is no longer detected by Drift Detection. Configuration drift arises when changes made externally affect remote objects in a manner that renders your infrastructure configuration invalid. State drift, on the other hand, emerges when modifications are applied to remote objects without invalidating your infrastructure configuration. See [Refresh-Only Mode](/terraform/cloud-docs/run/modes-and-options#refresh-only-mode) for more information about the steps required to remediate state drift.
```

Comment:
We _try_ not to use callouts unless absolutely necessary in the docs, for a better reading experience. This feels like good context, but not absolutely necessary. 

I tried to pare this down to the brass tacks, but please feel free to push back if this is not correct or doesn't feel good!
```suggestion
Configuration drift differs from state drift.  Drift detection _does not detect_ state drift.

Configuration drift happens when external changes affecting remote objects invalidate your infrastructure configuration. State drift occurs when external changes affecting remote objects _do not_ invalidate your infrastructure configuration. Refer to [Refresh-Only Mode](/terraform/cloud-docs/run/modes-and-options#refresh-only-mode)  to learn more about remediating state drift.
```